### ISSUE TITLE
Value of Text-Input not shown

### ISSUE THREAD

**Issue by marcobusemann** [directus/app#2807]
# Bug Report

## Steps to Reproduce

One of my collections has field ```title``` of type Text-Input.  
![image](https://user-images.githubusercontent.com/13619853/74276660-71145a80-4d16-11ea-83a4-55eba94b3ee1.png)

All items in that collection have a valid title set. But that title is not displayed in the list of collection items:
![image](https://user-images.githubusercontent.com/13619853/74276523-2d215580-4d16-11ea-81a9-aea41e442798.png)

If i create a new collection with a field of type Text-Input, it will work as expected.

## Expected Behavior

The title should be displayed.

## Actual Behavior

The title is not displayed in the list of collection items. When editing the item, the title is set correctly.

## Technical Details

- Device: Desktop
- OS: Windows 10
- Web Server:  Apache 2.4.37]
- PHP Version: [eg: 7.2.0]
- Database: [eg: MySQL 8.0.12]
- Directus Release Version: 8.5.4
- Install Method: Docker

**Comment by jpetryk** [comment]
I assume this is way too large of a change to expect a patch to make its way into v8? Great to hear it is fixed in V9, but is there an ETA on v9's official release date? Sorry if this information is available somewhere. Thanks!

**Comment by benhaynes** [comment]
Version 9 release is estimated at about 4-6 weeks... but that could flex depending on if we decide to add new optimizations/features, or move some to v9.x.

**Comment by rijkvanzanten** [comment]
@jpetryk Yeah it's not that easy to backport that into v8. v7/v8 were build on the idea of loading all needed data up front, and then have the interfaces only worry about displaying values. This works pretty great for most uses, except for deeper relational values where you hit a point where the nested content is missing.. In v9, I've restructured that so every level is "flat", and the relational interfaces themselves are in charge of fetching all the content they need to render out deep structures. This in turn means that we should never run into this situation again where content starts to go missing

**Comment by jpetryk** [comment]
@rijkvanzanten thanks for the extra bit of info. Are we just talking about changes in the display layer or does v9 come with a bunch of database migrations to adjust for this new flat structure. I ask because we're going to be building a base set of collections and fields that relate to one another (which we'll use to populate other projects) and it might be a bit risky to spend too much time doing so if they have to run through a bunch of migrations when we upgrade. Sorry for all the questions - I'm new to Directus but it looks like it will solve all our needs with the exception of this particular bug! Thanks again.

**Comment by rijkvanzanten** [comment]
Ohh no no, this is just how the admin app internally fetches / manages the data coming from the API. The IO of the API itself and how you configure relations isn't involved in any of that 🙂

**Comment by AAP-Becky** [comment]
I am experiencing the same thing in my interface

**Comment by ralcaidev** [comment]
Just used 8.8.0 from docker and I've found this problem.

Any news?

**Comment by ckyleda** [comment]
Is there any workaround for this as it's obviously a massive breaking problem?

I mean, to have relations broken in a CMS...

**Comment by fademar** [comment]
I am experiencing the same issue on one of my instance that runs Directus 8.8.1 version. An older one, 8.6.0, is free from this bug.
